lib/system.ml

```diff
@@ -302,7 +302,7 @@ let with_time ~batch f x =
     raise e
 
 let get_toplevel_path top =
-  let dir = Filename.dirname Sys.argv.(0) in
```

Comment:
This was broken when using Coq from PATH.

Reply:
It works like this [getting the absolute path] on purpose.

Reply:
argv.(0) is not the absolute path.